fw/cache.c

```diff
@@ -3159,7 +3189,8 @@ tfw_cache_start(void)
 		goto close_db;
 	}
 #endif
-	tfw_init_node_cpus();
+	if(tfw_init_node_cpus() == -ENOMEM) 
```

Comment:
```suggestion
	if (tfw_init_node_cpus() == -ENOMEM) 
```

Reply:
Why only on `-ENOMEM`? If we adjust `tfw_init_node_cpus()`, then we'll get a bug here

Reply:
What do you mean by the term 'adjust'?

Reply:
Perhaps, you should react on any error. That way you don't depend on possible future changes in the underlying function.